.gitlab/deps_build.yml

```diff
@@ -31,16 +31,12 @@
       -DLLVM_PARALLEL_LINK_JOBS=4 \
       ../llvm
     - cmake --build . --target install
-    - cd $DATADOG_AGENT_EMBEDDED_PATH
-    - rm -rf bin share libexec lib/clang lib/cmake lib/*.so*
-    - mkdir -p $CI_PROJECT_DIR/.tmp
-    - tar cvaf $CI_PROJECT_DIR/.tmp/clang-$ARCH-11.0.1.tar.xz .
-    - $S3_CP_CMD $CI_PROJECT_DIR/.tmp/clang-$ARCH-11.0.1.tar.xz $S3_PERMANENT_ARTIFACTS_URI/clang-$ARCH-11.0.1.tar.xz
+    - $S3_CP_CMD $DATADOG_AGENT_EMBEDDED_PATH/bin/clang $S3_PERMANENT_ARTIFACTS_URI/clang.${ARCH} 
```

Comment:
We should include the full version in the S3 path, so that when we upgrade versions we don't clobber old artifacts that could still be in use.